packit_service/service/api/utils.py

```diff
@@ -89,10 +91,38 @@ def get_sync_release_info(sync_release_model: SyncReleaseModel):
         "pr_id": sync_release_model.get_pr_id(),
         "issue_id": sync_release_model.get_issue_id(),
         "release": sync_release_model.get_release_tag(),
+        "anitya_version": sync_release_model.get_anitya_version(),
     }
 
     project = sync_release_model.get_project()
-    result_dict["repo_namespace"] = project.namespace if project else ""
-    result_dict["repo_name"] = project.repo_name if project else ""
-    result_dict["project_url"] = project.project_url if project else ""
+    result_dict.update(get_project_info(project))
+
+    return result_dict
+
+
+def get_project_info(project: Union[AnityaProjectModel, GitProjectModel]):
+    result_dict = {}
+
+    anitya_project_id = anitya_project_name = anitya_package = project_url = (
+        repo_name
+    ) = repo_namespace = None
```

Comment:
A nitpick, but this is really ugly :sweat_smile: How about this?

```suggestion
    anitya_project_id = anitya_project_name = anitya_package = None
    project_url = repo_name = repo_namespace = None
```